jira-epic-stats.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Script to generate a report of all epics and their epic link counts in Jira
+# GitHub has a limit of 100 sub-issues per parent so best to adjust these in Jira prior to migration
+# Usage: ./jira-epic-stats.sh
+
+# Load environment variables
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+if [ -f "$SCRIPT_DIR/.env" ]; then
+  source "$SCRIPT_DIR/.env"
+fi
+
+# Validate required environment variables
+: "${JIRA_MIGRATION_JIRA_PROJECT_NAME:? Missing Jira project name (e.g., JENKINS)}"
+: "${JIRA_MIGRATION_JIRA_URL:? Missing Jira base URL (e.g., https://issues.jenkins.io)}"
+: "${JIRA_MIGRATION_JIRA_TOKEN:? Missing Jira token}"
+: "${JIRA_MIGRATION_PARALLEL_COUNT:=100}"
+
+# Check for required commands
+if ! command -v flock >/dev/null 2>&1; then
```

Comment:
Nit: can you ask your AI to use `parallel` like in check-email-domains.sh instead of `flock`?

Reply:
I could but its already in the two retrieve scripts watchers and remote links so its consistent with them and I assume why it used it.